src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -42,7 +42,6 @@ public PythonEngineVersion Engine
                 {
                     engine = value;
                     RaisePropertyChanged(nameof(Engine));
-                    OnNodeModified();
```

Comment:
I'm afraid this may have unforeseen consequences. Should we check all places where we are setting this property and make sure we are doing what we want to do (trigger or not).

Reply:
Engine is a relatively new property only having been introduced since the python 3 changes were brought in - *I think*.